cookbook/core/type_system/schema.py

```diff
@@ -9,6 +9,12 @@
 converted into flyte's abstract representation of a schema object
 
 """
+
+# %%
+# .. warning::
+#
+#   ``FlyteSchema`` is deprecated, use :ref:`structured_dataset_example` instead.
```

Comment:
can we mention when this will be deprecated? I think we decided 12 releases from now, which probably means `1.25.0`, unless we release `2.0.0` before that.